content/projects/basic-unit-test-mocks-kata/_index.md

```diff
@@ -180,4 +180,19 @@ As another example, if you were developing a "forgot password" or "confirm email
 - You can also add extra functionality. Eg: send an inspirational quote and a funny quote.
 - You can make the emails that get sent look good as well. it doesn't just need to be plain text
 - You can look into sending sms instead of email
-- link to the quote author's wikipedia page straight from the email/sms
\ No newline at end of file
+- link to the quote author's wikipedia page straight from the email/sms
+- add extra information such as a weather report
+
+
+**Instructions for reviewer**
```

Comment:
Please use heading syntax to show that something is a heading.  

This should probably have 2 hash symbols. Please look into markdown syntax and apply it consistently.